docs/reference/commands/node-tool.asciidoc

```diff
@@ -2,14 +2,15 @@
 == elasticsearch-node
 
 The `elasticsearch-node` command enables you to perform unsafe operations that
-risk data loss but which may help to recover some data in a disaster.
+risk data loss but which may help to recover some data in a disaster and
+repurpose nodes into new roles.
```

Comment:
Suggest breaking into two simpler sentences:

```
The `elasticsearch-node` command enables you to perform certain unsafe
operations on a node that are only possible while it is shut down. This command
allows you to adjust the <<modules-node,role>> of a node and may be able to
recover some data after a disaster.
```

Which reminds me, the docs about roles are at `docs/reference/modules/node.asciidoc` and I think we should have docs there to say that you now have to take special steps to change the role of a node.